documentation/modules/security/con-securing-kafka-authentication.adoc

```diff
@@ -95,6 +95,10 @@ mTLS listener configuration in the `Kafka` resource requires the following:
 * `tls: true` to specify TLS encryption and server authentication
 * `authentication.type: TLS` to specify the client authentication
 
+When a Kafka cluster is created by the Cluster Operator, it creates a new secret with the name `<cluster_name>-cluster-ca-cert`.
+The secret contains a private and public key for mTLS.
+The cluster secret xref:certificates-and-secrets-formats-{context}[provides keys and certificates in PEM and PKCS #12 formats].
```

Comment:
The public key - which is exected to be used by the users - is in PEM and PKCS12 formats. The private key is in PEM format only. In general, the private key should not be touched by user. So maybe you can just leave it out from this paragraph.